src/huggingface_hub/_space_api.py

```diff
@@ -118,3 +121,31 @@ def __init__(self, data: Dict) -> None:
         self.sleep_time = data["gcTimeout"]
         self.storage = data["storage"]
         self.raw = data
+
+
+@dataclass
+class SpaceVariable:
+    """
+    Contains information about the current variables of a Space.
+
+    Args:
+        key (`str`):
+            Variable key. Example: `"MODEL_REPO_ID"`
+        value (`str`):
+            Variable value. Example: `"the_model_repo_id"`.
+        description (`str` or None):
+            Description of the variable. Example: `"Model Repo ID of the implemented model"`.
+        updatedAt (`datetime`):
+            datetime of the last update of the variable.
+    """
+
+    key: str
+    value: str
+    description: Optional[str]
+    updatedAt: datetime
+
+    def __init__(self, k, v: Dict) -> None:
+        self.key = k
+        self.value = v["value"]
+        self.description = v["description"]
+        self.updatedAt = parse_datetime(v["updatedAt"])
```

Comment:
```suggestion
    def __init__(self, key: str, values: Dict) -> None:
        self.key = key
        self.value = values["value"]
        self.description = values.get("description")
        self.updated_at = parse_datetime(v["updatedAt"])
```

- `description` is optional. If it is not returned by the server, we should set it to None
- best to explicit the parameters (k => key, v => values) + best to explicit the key type (str)
- updatedAt => updated_at

Reply:
The `values.get("description")` produces an exception for me.

I'll change it to:
```
if "description" in values.keys():
            self.description = values["description"]
```

Reply:
Hmmm, which error are you getting with `values.get("description")` ? Your solution is valid as well (you can even do `if "description" in values:` instead) but using `.get(...)` should be equivalent.

Reply:
A good question. I just tried to replicate it and somehow can't. 
I must have made an error previously.

Changed to your proposal of `values.get("description")` because it's less code.